### PR TITLE
MUSIC OVERHAUL + NEW THEMES!

### PR DESCRIPTION
**Pull request by yellowface7**
New music! Hooray!

+New lavaland music to remind the miners that they are in literal hell.
+New spess music!!
+NEW THEMES **OH MY FUCKING GOD**

-old theme

This whole pr took me about a month and a half to do. More prs from me is on the way!

### CODE REVIEW

File: code/__DEFINES/sound.dm [hunk 1]
```diff
@@ -46,14 +46,16 @@
 #define MINING list('sound/ambience/ambimine.ogg', 'sound/ambience/ambicave.ogg', 'sound/ambience/ambiruin.ogg',\
 											'sound/ambience/ambiruin2.ogg',  'sound/ambience/ambiruin3.ogg',  'sound/ambience/ambiruin4.ogg',\
 											'sound/ambience/ambiruin5.ogg',  'sound/ambience/ambiruin6.ogg',  'sound/ambience/ambiruin7.ogg',\
-											'sound/ambience/ambidanger.ogg', 'sound/ambience/ambidanger2.ogg', 'sound/ambience/ambimaint1.ogg', 'sound/ambience/ambilava.ogg')
+											'sound/ambience/ambidanger.ogg', 'sound/ambience/ambidanger2.ogg', 'sound/ambience/ambimaint1.ogg', 'sound/ambience/ambilava.ogg',\
```
Comment (ghost) [inline code/__DEFINES/sound.dm:49]: add a `// sunset -- add new ambience`

File: code/__DEFINES/sound.dm [hunk 2]
```diff
@@ -46,14 +46,16 @@
...
+											'sound/ambience/ambiruin8.ogg', 'sound/ambience/ambiruin9.ogg', 'sound/ambience/ambiruin10.ogg', 'sound/ambience/ambiruin11.ogg',\
+											'sound/ambience/ambiruin12.ogg', 'sound/ambience/ambiruin13.ogg', 'sound/ambience/ambiruin14.ogg', 'sound/ambience/ambiruin15.ogg'\)
 
 #define MEDICAL list('sound/ambience/ambinice.ogg')
 
 #define SPOOKY list('sound/ambience/ambimo1.ogg','sound/ambience/ambimo2.ogg','sound/ambience/ambiruin7.ogg','sound/ambience/ambiruin6.ogg',\
 										'sound/ambience/ambiodd.ogg', 'sound/ambience/ambimystery.ogg')
 
-#define SPACE list('sound/ambience/ambispace.ogg', 'sound/ambience/ambispace2.ogg', 'sound/ambience/title2.ogg', 'sound/ambience/ambiatmos.ogg')
+#define SPACE list('sound/ambience/ambispace.ogg', 'sound/ambience/ambispace2.ogg', 'sound/ambience/ambispace3.ogg', 'sound/ambience/ambispace4.ogg', 'sound/ambience/ambiatmos.ogg')
```
Comment (ghost) [inline code/__DEFINES/sound.dm]: add a `// sunset -- add new ambience`

**Review by ghost — CHANGES_REQUESTED** [review]
Move the music files into `sunset/sound/...`

File: code/__DEFINES/sound.dm (hunk 2 above)
Comment (yellowface7) [reply code/__DEFINES/sound.dm]: Hm? Is that for logging reasons?

**Comment by FlattestGuitar** [comment]
None of the music has any credit to the authors attached, and it sure as heck doesn't look like you gave a damn about having permission to use it in the first place.

Also, I really don't think most of those fit, but I couldn't be bothered to listen to all of them.

**Review by alexkar598 — DISMISSED** [review]
Either you forgot to upload ambiruin13 to ambiruin15 or you fucked up

Also give credits

**Comment by yellowface7** [comment]
Ah. I may have fucked up. I will be sure to give credits

**Comment by yellowface7** [comment]
Also, most of the ambiruin music were for lavalands. I wanted to remind the miners that they are in hell.

**Comment by yellowface7** [comment]
+Credits to all of the audio

-ambiruin13
-ambiruin14      } All of these were of human error. I fucked up at these ones.
-ambiruin15

Lavalands will have a different theme than the rest of the server since lavalands itself has a different theme. I added all of these songs to the lavalands to remind the miners that they are in hell and to add some more hellscape flavor to the lavalands.

**Comment by miniusAreas** [comment]
@alexkar598 @steamp0rt Fix your stale reviews, for the sake of sanity, please.

**Comment by miniusAreas** [comment]
Also @yellowface7, congrats on this, hope to see it merged. Good luck on the -html NTSL, and the telecoms shenanigins that require me to remap parts of the station :p

**Review by ghost — CHANGES_REQUESTED** [review]
Defines need to be modularized

**Comment by ghost** [comment]
Inactive, not modularized

**Comment by yellowface7** [comment]
UPDATE: I am currently working on modulating the stuff. I am currently running into errors.

**Comment by yellowface7** [comment]
COMMIT HIGHLIGHTS:

+Modularization.
+Title music is quieter
+Added titles to the music queue

-Obnoxiously loud music

**Comment by yellowface7** [comment]
Okay. Plans are in for the final push. It will go in effect either tonight or tomorrow so keep an eye out. I will ree if it doesn't push right.
=

**Comment by yellowface7** [comment]
**Update:** Github is crapping out right now. I accidentally deleted my repo on my computer and github won't let me redownload it because it's going through puberty or something. As soon as I am allowed to download it, I will and I _will_ make those final changes.

Also, keep an eye out for my second pr, it will port the floor cluwne over to this server. I am almost done with it. I am currently working on the final seven errors which are pretty big ones. Hopefully that pr won't take nearly as long as this one to complete.

**Comment by yellowface7** [comment]
Still having issues with downloading the branch.

**Comment by yellowface7** [comment]
Finally, after all of this work, it should be ready to be rolled out. If there is anything I missed, please tell me.

**Review by ghost — APPROVED** [review]

**Pull request merged**